### DISCUSSION TITLE
Could you create a table of contents function that automatically follows the headings?

### DISCUSSION THREAD

**Discussion by eunfngswr** [zauberzeug/nicegui#1733] [Ideas / Feature Requests]
I would like you to make a table of contents feature that automatically follows like the one implemented on the right side of https://squidfunk.github.io/mkdocs-material/getting-started/

**Comment by falkoschindler** [comment]
Hi @eunfngswr, this looks like a duplicate of https://github.com/zauberzeug/nicegui/discussions/1592.
Let me know if I'm mistaken.

**Reply by eunfngswr** [reply]
Thank you for the reply. I think it is the same as [#1592 .](https://github.com/zauberzeug/nicegui/discussions/1592) So I have to implement it myself and it is not officially supported? I hope it will be officially supported.
